packages/block-library/src/index.js

```diff
@@ -135,4 +136,8 @@ export const registerCoreBlocks = () => {
 		setFreeformContentHandlerName( classic.name );
 	}
 	setUnregisteredTypeHandlerName( missing.name );
+
+	if ( group ) {
```

Comment:
When would this not be true?

Reply:
I was browsing the file and came to ask the same thing. :) @getdave

Reply:
Pull request at #22563